service/prometheus/prometheus.go

```diff
@@ -1,6 +1,27 @@
 package prometheus
 
+import (
+	"strings"
+
+	"k8s.io/client-go/pkg/api/v1"
+)
+
 const (
 	CertificateAnnotation = "giantswarm.io/prometheus-config-controller/certificate"
 	ClusterAnnotation     = "giantswarm.io/prometheus-config-controller/cluster"
 )
+
+// GetCertificateName returns the value of the certificate annotation, splitted by '/'.
+// Assumed that the service contains this annotation, see `FilterInvalidServices`.
+func GetCertificateName(service v1.Service) (string, string) {
```

Comment:
I can see this logic being reused, so i'm putting it into `prometheus` package a little early.

Reply:
We have put such things into a `key` package.

Reply:
I prefer it in `prometheus`, as it means we should be able to keep the annotation details internal to the package. I thought about moving to `key`, but that still leaks the annotation details.